libs/sdk-core/src/lsp.rs

```diff
@@ -74,7 +67,7 @@ impl LspAPI for BreezServer {
         let response = client.lsp_list(request).await?;
         let mut lsp_list: Vec<LspInformation> = Vec::new();
         for (key, value) in response.into_inner().lsps.into_iter() {
-            lsp_list.push(convert_to_lsp_info(key, value));
+            lsp_list.push(convert_to_lsp_info(key, value)?);
```

Comment:
Now that this may fail with error we should not let this affect other lsps. I believe we should log the error and keep trying to add the rest of the lsps the list.

Reply:
I had to revert back to non-fallible conversion, so this is not a problem anymore.